tests/MongoDB.Driver.Tests/JsonDrivenTests/JsonDrivenCollectionTest.cs

```diff
@@ -15,35 +15,57 @@
 
 using System.Collections.Generic;
 using MongoDB.Bson;
+using MongoDB.Bson.TestHelpers.JsonDrivenTests;
 
 namespace MongoDB.Driver.Tests.JsonDrivenTests
 {
-    public abstract class JsonDrivenCollectionTest : JsonDrivenDatabaseTest
+    public abstract class JsonDrivenCollectionTest : JsonDrivenCommandTest
```

Comment:
This class is the base class for all tests where _collection is the receiver of the method call.